src/components/analytics-shell/analytics-shell.js

```diff
@@ -59,6 +62,12 @@ AnalyticsShell.defaultProps = {
   sentry: {
     dsn: 'https://6ba8cfeeedad4fb7acb8576f0fd6e266@sentry.io/1384508'
   },
+  sentryPerformance: {
+    // This enables automatic instrumentation (highly recommended)
+    integrations: [new TracingIntegrations.BrowserTracing()],
+    // To set a uniform sample rate
+    tracesSampleRate: 0.2
```

Comment:
What do you think about making the sampling rate configurable in case we need to tweak for various sites without doing a dr-ui release? The [Sentry docs](https://docs.sentry.io/platforms/javascript/performance/#verify) recommend testing with a rate of 1 so we may at least want to cover that use case. What do you think?

Reply:
Per chat, these value are configurable. This array represents the default prop.